CMakeLists.txt

```diff
@@ -87,6 +87,10 @@ set(WXFB_GENERATOR_LANGUAGES cpp python lua php CACHE INTERNAL "")
 
 if(NOT WXFB_WXWIDGETS_ENABLE_30)
   set(wxWidgetsVersion 3.1.2)
+
+  if(MSYS)
+    set(ENV{WX_CONFIG} "wx-config-3.1")
+  endif(MSYS)
```

Comment:
The purpose of these environment variables is that users can give information to CMake that is required to build on their machine without modifying the CMake files of the project. Now if you forcefully set this variable this isn't possible anymore. I don't think this variable should be set here, it should be set by external means (i am using MSVC and have setup scripts that prepare the environment, at least on this platform this is kind of a common approach).

If you really think this should be set in the project file, then please check if this environment variable is already set and don't touch it in that case.